content/learn/book/getting-started/ecs/_index.md

```diff
@@ -117,11 +117,11 @@ fn greet_people(query: Query<&Name, With<Person>>) {
 }
 ```
 
-The parameters we pass in to a "system function" define what data the system runs on. In this case, `greet_people` will run on all entities with the `Person` and `Name` component.
+The parameters we pass into a _system function_ define what data the system runs on. In this case, `greet_people` will run on all entities with the `Person` and `Name` component.
```

Comment:
As a general style thing we don't use italics to indicate "concepts". We use quotes for that. Italics are for emphasis.